src/vs/workbench/contrib/chat/browser/chatEditing/chatEditingActions.ts

```diff
@@ -98,6 +98,11 @@ registerAction2(class RemoveFileFromWorkingSet extends WorkingSetAction {
 				order: 5,
 				group: 'navigation'
 			}],
+			keybinding: {
+				primary: KeyCode.Delete,
+				when: ContextKeyExpr.and(ChatContextKeys.location.isEqualTo(ChatAgentLocation.EditingSession), ChatContextKeys.inChatInput.negate()),
```

Comment:
I believe we need some kind of context key that expresses "working set focused" because otherwise the keybinding is active for the conversation etc pp. So, instead of spelling out the "not here places, like inChatInput.negate" we need to know that the working set list is active